scalableminds/webknossos
# Docker healthchecks

- Adds healthchecks to all dockerfiles for auto-healing and starting order.
- Uses the new integrated fossildb healthcheck https://github.com/scalableminds/fossildb/pull/30

### Steps to test:
- `DOCKER_TAG=docker-health__7392 docker-compose pull webknossos webknossos-datastore webknossos-tracingstore`
- `DOCKER_TAG=docker-health__7392 docker-compose up webknossos webknossos-datastore webknossos-tracingstore`
- Check `docker ps` whether all services are reported healthy (after ~1min each)

### Issues:
- fixes #3569 

------
~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~
~- [ ] Updated [documentation](../blob/master/docs) if applicable~
